Epsilon.Host.Frontend/src/api.generated.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable */
 /* tslint:disable */
+// prettier-ignore
```

Comment:
Added prettier ignore, So that prettier does not change names and values in a generated file